### DISCUSSION TITLE
Hello. Does the "Offset" means Cacheline offeset?  Does the "CLOffset" means byte offset?

### DISCUSSION THREAD

**Discussion by szuhzw** [CMU-SAFARI/Hermes#15] [General]
![Uploading image.png…]()

**Comment by szuhzw** [comment]
![image](https://github.com/CMU-SAFARI/Hermes/assets/65290731/3b5b4384-6fd8-41f3-9fd2-2496680f03d8)

**Comment by rahulbera** [comment]
CLoffset is the byte offset of the data within the cacheline. Please see lines 194-200 in `src/offchip_pred_perc.cc` to understand how the features are getting calculated.

Please create a post in the discussion section of the repository (https://github.com/CMU-SAFARI/Hermes/discussions) for further questions.

**Reply by szuhzw** [reply]
Is my understanding correct that CLOffset represents the word offset? @rahulbera 
![image](https://github.com/user-attachments/assets/a849e3a4-2f7a-4753-8446-9d3834288072)
![1720954284530](https://github.com/user-attachments/assets/417c8213-6a20-451f-87e9-2be32e321574)
![1720954284532](https://github.com/user-attachments/assets/a3cc4c16-0d83-4a7b-9d85-2e57ef7a5c58)

**Reply by szuhzw** [reply]
Hello. @rahulbera
